style.css

```
@@ -1,20 +1,18 @@
 * {
     box-sizing: border-box;
     margin: 0;
-    font-family: 'Pretendard';
+    font-family: "Pretendard Variable", Pretendard, -apple-system, BlinkMacSystemFont, system-ui, Roboto, "Helvetica Neue", "Segoe UI", "Apple SD Gothic Neo", "Noto Sans KR", "Malgun Gothic", "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif;
 }
 
-@media (max-width: 800px) {
-    .small-tomato {
-      background-color: tomato;
-    }
-  }
-
-body {
-    width: 100%;
-    height: 100%;
-    text-decoration-line: none;
+header {
```

❗️ 수정요청
`아래로 스크롤 해도 상단 네비게이션 바(Global Navigation Bar)가 최상단에 고정됩니다.` 요구사항이 안 구현된 것 같아요~

```suggestion
header {
  position: sticky;
  top: 0;
```

위와 같이 추가해서 header가 스크롤이 내려가도 고정되게 해주세요~